.travis.yml

```diff
@@ -8,11 +8,14 @@ jdk: openjdk8
 
 before_install: "cp .maven.settings.xml $HOME/.m2/settings.xml"
 
+script: "mvn cobertura:cobertura"
```

Comment:
It might be more preferable to execute it via the `verify` goal rather than cobertura:cobertura

Reply:
This is how the example says to set it up